card/src/main/res/values-cs-rCZ/strings.xml

```diff
@@ -12,11 +12,18 @@
     <string name="checkout_card_expiry_date_hint">Konec platnosti</string>
     <string name="checkout_card_security_code_hint">Kód CVC/CVV</string>
     <string name="checkout_card_security_code_optional_hint">Kód CVC/CVV (volitelné)</string>
+    <string name="checkout_social_security_number_hint"></string>
```

Comment:
Was the social_security_number change on purpose? Idk if our string generator project handles translatable=false correctly.

Reply:
it wasn't, reverted it.